exporter/prometheusremotewriteexporter/config.go

```diff
@@ -64,6 +66,16 @@ type RemoteWriteQueue struct {
 	NumConsumers int `mapstructure:"num_consumers"`
 }
 
+var dropSanitizationGate = featuregate.Gate{
+	ID:          "exporter.prometheusremotewrite.SanitizeLabel",
+	Enabled:     true,
```

Comment:
```suggestion
	ID:          "exporter.prometheusremotewrite.PermissiveLabelSanitizatin",
	Enabled:     false,
```

Feature gates should start disabled by default, eventually transition to enabled by default if the feature is working well, and then finally be removed entirely when the feature is always enabled and users can no longer opt out.

Reply:
updated